src/test/java/racingcar/dao/GameJdbcDaoTest.java

```diff
@@ -26,15 +30,14 @@ void setUp() {
     }
 
     @Test
-    void 게임을_저장한다() {
+    void saveAndGetId_메서드는_게임을_저장하고_저장한_게임의_id값을_반환한다() {
         // given
-        final int trialCount = 5;
-        final String winners = "비버";
+        final GameEntity game = new GameEntity(5);
 
         // when
-        final int id = gameDao.save(trialCount, winners);
+        final Optional<Integer> gameId = gameDao.saveAndGetId(game);
 
         // then
-        assertThat(id).isPositive();
+        assertThat(gameId.isPresent()).isTrue();
```

Comment:
Optional의 존재를 검사하는 isPresent()를 활용해보면 어떨까요?😉
```suggestion
        assertThat(gameId).isPresent();
```

Reply:
Optional 검사할 때 isPresent를 사용할 수 있군요! 처음 알았습니다. 👍👍👍